src/components/html/EcAppCard.html

```diff
@@ -3,6 +3,9 @@
   style="cursor: pointer"
   @click="$emit('click')"
 >
+  <div class="outdated-label alert alert-warning" v-if="newVersionAvailable">
```

Comment:
```suggestion
  <div
    v-if="newVersionAvailable"
    class="w-100 position-absolute alert alert-warning"
  >
```

Em apps como o admin marketplace que não precisamos nos preocupar com customização de tema pelo lojista ou parceiro, é melhor usar utility classes ou até mesmo inline styles (quando não houver classe utilitária), simplifica...